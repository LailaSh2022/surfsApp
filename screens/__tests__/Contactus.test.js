import React from "react";
import renderer from "react-test-renderer";
import ContactUs from "./../ContactUs";

describe("ContactUs component", () => {
  test("Matches the snapshot", () => {
    const tree = renderer.create(<ContactUs />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
