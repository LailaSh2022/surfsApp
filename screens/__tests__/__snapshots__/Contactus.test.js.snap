// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactUs component Matches the snapshot 1`] = `
<View
  style={
    [
      {
        "backgroundColor": "#ffffff",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "paddingBottom": 25,
        "paddingLeft": 25,
        "paddingRight": 25,
        "paddingTop": 64,
      },
    ]
  }
>
  <View
    style={
      [
        {
          "alignItems": "center",
          "flexBasis": 0,
          "flexGrow": 1,
          "flexShrink": 1,
          "width": "100%",
        },
      ]
    }
  >
    <View
      style={
        {
          "marginTop": "-20%",
          "top": 0,
        }
      }
    >
      <Image
        resizeMode="cover"
        source={1}
        style={
          [
            {
              "height": 170,
              "width": 200,
            },
          ]
        }
      />
    </View>
    <Text
      style={
        [
          {
            "color": "#152A5F",
            "fontSize": 30,
            "fontWeight": "bold",
            "paddingBottom": 10,
            "paddingLeft": 10,
            "paddingRight": 10,
            "paddingTop": 10,
            "textAlign": "center",
          },
        ]
      }
    >
      Contact Us
    </Text>
    <RCTScrollView>
      <View>
        <Image
          style={
            {
              "borderRadius": 5,
            }
          }
        />
        <View
          style={
            [
              {
                "alignItems": "center",
                "flexDirection": "row",
                "justifyContent": "center",
                "paddingBottom": 10,
                "paddingLeft": 10,
                "paddingRight": 10,
                "paddingTop": 10,
              },
            ]
          }
        >
          <View
            style={
              {
                "flexDirection": "column",
              }
            }
          >
            <Text
              style={
                [
                  {
                    "alignItems": "center",
                    "color": "#152A5F",
                    "fontSize": 15,
                    "justifyContent": "center",
                  },
                ]
              }
            >
              Name: SurfsApp Limited 
            </Text>
            <Text
              style={
                [
                  {
                    "alignItems": "center",
                    "color": "#152A5F",
                    "fontSize": 15,
                    "justifyContent": "center",
                  },
                ]
              }
            >
              Phone Number: 09 827 6100 
            </Text>
            <Text
              style={
                [
                  {
                    "alignItems": "center",
                    "color": "#152A5F",
                    "fontSize": 15,
                    "justifyContent": "center",
                  },
                ]
              }
            >
              Address: 242 Queen Street, Auckland CBD 
            </Text>
            <Text
              style={
                [
                  {
                    "alignItems": "center",
                    "color": "#152A5F",
                    "fontSize": 15,
                    "justifyContent": "center",
                  },
                ]
              }
            >
              Auckland 1010 
            </Text>
            <Text
              style={
                [
                  {
                    "alignItems": "center",
                    "color": "#152A5F",
                    "fontSize": 15,
                    "justifyContent": "center",
                  },
                ]
              }
            >
              Email: surfsapp.limited@gmail.com 
            </Text>
            <Text
              style={
                [
                  {
                    "alignItems": "center",
                    "color": "#152A5F",
                    "fontSize": 15,
                    "justifyContent": "center",
                  },
                ]
              }
            >
              Opening Hours: 9:00am - 5:00 pm 
            </Text>
          </View>
        </View>
      </View>
    </RCTScrollView>
  </View>
</View>
`;
